### ISSUE TITLE
why is my dwc:establishmentMeans discarded?

### ISSUE THREAD

**Issue by Jegelewicz** [gbif/portal-feedback#4885]
https://www.gbif.org/occurrence/1323885367

is sent to the IPT with 'captive' in dwc:establishmentMeans.

![image](https://github.com/gbif/portal-feedback/assets/5725767/e5469ca4-1af2-4cc4-8681-c20b7b7b2423)

BUT it is excluded from the GBIF data.

Why is this? the Darwin Core Quick Reference clearly shows 'captive' as an example for this term - https://dwc.tdwg.org/terms/#dwc:degreeOfEstablishment

It seems to me that assisting those downloading data from GBIF to remove 'captive' records or to use only 'captive' records would be an important objective. There is a place to provide this information, but it is being removed by GBIF.

If dwc:establishmentMeans is not the correct field for this information in the GBIF DwC-A, where should it be placed?

**Comment by MortenHofft** [comment]
TLDR: it isn't removed but provided in the wrong field.

`degree of establishment` and `establishment means` are 2 different fields.

establishmens means do not have captive as an example https://dwc.tdwg.org/list/#dwc_establishmentMeans

`establishment means` is handled as a vocabulary in GBIF, meaning that different spellings, languages etc are mapped to one vocabulary. This is done to help with search. occasionally it will also mean that meaningful values are lost, but we have the means to quickly correct that by adding alternative labels (e.g. `Introducida (Colonización asistida)`)

`degreeOfEstablishment` is also a vocabulary. You can see the concepts here
https://registry.gbif.org/vocabulary/DegreeOfEstablishment/concepts

Captive is indeed one of them.

Did that answer your question?

**Comment by Jegelewicz** [comment]
Thank you for pointing out my confusion!
